Makefile

```diff
@@ -25,17 +25,17 @@ SUSE_STYLES_PATH := $(DB_XML_PATH)/stylesheet
 
 NOVDOC_NAME     := novdoc
 NOVDOC_VERSION  := 1.0
-NOVDOC_DTD_PATH := $(SUSE_SCHEMA_PATH)/dtd/$(NOVDOC_VERSION)
+NOVDOC_DTD_PATH := $(SUSE_SCHEMA_PATH)/NovDoc/dtd/$(NOVDOC_VERSION)
 
 #--------------------------------------------------------------
-# SUSEDOC
+# GEEKODOC
 
-#SUSEDOC_NAME     := susedoc
-#SUSEDOC_VERSION  := 0.9
-#SUSEDOC_RNG_PATH := $(SUSE_SCHEMA_PATH)/rng/$(SUSEDOC_VERSION)
+#GEEKODOC_NAME     := geekodoc
```

Comment:
If you want to use these names in the future, maybe use a more general name like `SUSESCHEMA`? Something I did in my Makefile for GeekoDoc. ;-)